main.py

```diff
@@ -0,0 +1,18 @@
+from fastapi import FastAPI
+
+from weather.router import router as cities_router
+from temperature.router import router as temperature_router
+
+app = FastAPI()
+
+app.include_router(
+    router=cities_router, prefix="/weather", tags=["Cities"]
+)
+app.include_router(
+    router=temperature_router, prefix="/weather", tags=["Temperature"]
```

Comment:
Both routers are prefixed with `/weather` and have overlapping tags ('Cities' and 'Temperature'). Consider using distinct prefixes or tags to avoid potential confusion in the API documentation.